### PR TITLE
Unable to find the output in the server log

### LINKED ISSUE #19138: Unable to find the output in the server log
Despite the fact that "Unable to find the output in the server log" exception is created, searched String appears in the log later. In order to see PreDestroy message, a wait is added till application to stop.

### PR DESCRIPTION
**Pull request by BerksanAtes**
Fixes #19138

### CODE REVIEW

**Comment by BerksanAtes** [comment]
SOE Build -> https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_nniFsI3SEeyCwKQyYOIq9g

File: dev/io.openliberty.ws.jaxws.global.handler.internal_fat/fat/src/com/ibm/ws/webservices/handler/fat/HandlerChainTest.java [hunk 1]
```diff
@@ -115,8 +115,11 @@ public void testProviderHandlerChainWithGlobalHandlers() throws Exception {
             throw se;
         }
         // Uninstall Applications
-        server.deleteAllDropinApplications();
-        
+        server.removeAndStopDropinsApplications("testHandlerProvider.war","testHandlerClient.war","testHandlerClientWithoutXML.war");
```
Comment (Azquelt) [inline dev/io.openliberty.ws.jaxws.global.handler.internal_fat/fat/src/com/ibm/ws/webservices/handler/fat/HandlerChainTest.java]: `removeAndStopDropinsApplications` returns `true` if the applications stopped and `false` if they didn't, so you probably want to `assertTrue` around this method.

Comment (BerksanAtes) [reply dev/io.openliberty.ws.jaxws.global.handler.internal_fat/fat/src/com/ibm/ws/webservices/handler/fat/HandlerChainTest.java]: Great suggestion!

**Review by neuwerk — APPROVED** [review]

**Comment by BerksanAtes** [comment]
#run-libby-bot

**Pull request merged**